CHANGELOG.md

```diff
@@ -16,7 +17,7 @@
 - Bump Native SDK to v0.4.18 ([#810](https://github.com/getsentry/sentry-unity/pull/810) & [#824](https://github.com/getsentry/sentry-unity/pull/824))
   - [changelog](https://github.com/getsentry/sentry-native/blob/master/CHANGELOG.md#0418)
   - [diff](https://github.com/getsentry/sentry-native/compare/0.4.15-7-g9eecb1b...0.4.18)
-- Bump CLI to v2.2.0 ([#826](https://github.com/getsentry/sentry-unity/pull/826))
+- Bump Sentry CLI to 2.2.0 ([#826](https://github.com/getsentry/sentry-unity/pull/826))
```

Comment:
@bitsandfoxes 

1. We don't say "sentry native SDK", "Sentry Java SDK" so why Sentry CLI?
2. Same about the `v` prefix, which is there intentionally here and in other changelog entries above

Reply:
You're right, I got swept up by the moment. I'll correct it.